VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -339,7 +345,9 @@ export abstract class WaitingRoomBaseComponent {
         this.errorCount++;
         this.connected = false;
         this.updateShowVideo();
-        this.logger.error(`[WR] - Error from pexip. Reason : ${error.reason}`, new Error(error.reason), { pexipError: error });
+        this.logger.error(`${this.loggerPrefix} Error from pexip. Reason : ${error.reason}`, new Error(error.reason), {
```

Comment:
could you add the conference and participant as the third param like the other logs do please

Reply:
added conference id and participant id